src/app-bar.jsx

```diff
@@ -247,6 +253,8 @@ const AppBar = React.createClass({
       style: prepareStyles(Object.assign(styles.title, styles.mainElement, titleStyle)),
     }, title);
 
+    const iconLeftStyle = Object.assign({}, styles.iconButtonStyle, iconStyleLeft);
```

Comment:
you shouldn't do this either as it's resulting in calling prepareStyles twice. Please follow my comments as they are :grin: